packages/notifications/resources/views/components/database/modal/index.blade.php

```diff
@@ -29,7 +29,6 @@
                     'dark:border-gray-700' => ! $notification->unread(),
                 ])>
                     <div @class([
-                        'py-2 pl-4 pr-2',
```

Comment:
What's this change for?

Reply:
This had to be removed to allow the background color to go all the way to the edge.

Reply:
Got it.